packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/google-drive.tsx

```diff
@@ -0,0 +1,87 @@
+import classnames from 'classnames';
+import React from 'react';
+import { Button, Icon, StaticUrl, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+
+type TGoogleDrive = {
+    is_authorised: boolean;
+    is_open_button_loading: boolean;
+    onDriveConnect: () => void;
+    onDriveOpen: () => void;
+    is_mobile: boolean;
+};
+
+const GoogleDrive = ({
+    is_authorised,
+    is_open_button_loading,
+    onDriveConnect,
+    onDriveOpen,
+    is_mobile,
+}: TGoogleDrive) => {
+    return (
+        <div className='load-strategy__container'>
+            <div className='load-strategy__google-drive'>
+                <Icon
+                    icon={'IcGoogleDrive'}
+                    className={classnames('load-strategy__google-drive-icon', {
+                        'load-strategy__google-drive-icon--disabled': !is_authorised,
+                    })}
+                    size={is_mobile ? 96 : 128}
+                />
+
+                <div className='load-strategy__google-drive-text'>
+                    {is_authorised ? (
+                        <Localize i18n_default_text='You are connected to Google Drive' />
+                    ) : (
+                        'Google Drive'
+                    )}
+                </div>
+                <Text
+                    as='p'
+                    align='left'
```

Comment:
we can remove this line since `left` is the default value

Reply:
done ✅